direct/data/datasets_config.py

```diff
@@ -38,6 +38,8 @@ class RandomAugmentationTransformsConfig(BaseConfig):
     random_flip: bool = False
     random_flip_type: Optional[str] = "random"
     random_flip_probability: Optional[float] = 0.5
+    random_reverse: bool = False
+    random_reverse_probability: Optional[float] = 0.5
```

Comment:
Why did you not choose for "False" to disable this instead of a second variable?

Reply:
Now `random_reverse_probability` controls it.